sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantPreAuthSettings.md

```diff
@@ -0,0 +1,47 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spotenantpreauthsettings
+applicable: SharePoint Online
+title: Get-SPOTenantPreAuthSettings
+schema: 2.0.0
+author: lw-msft
+ms.author: laurenwong
+ms.reviewer:
+manager: bhaveshd
+---
+
+# Get-SPOTenantPreAuthSettings
+
+## SYNOPSIS
+
+Gets the pre auth settings for the tenant.
```

Comment:
Add a brief sentence that explains what "pre auth" is. Alternatively, if pre auth is already documented somewhere else in learn.microsoft.com, link it here:

```
See the [pre auth documentation](xxx) for more information.
```